src/benchq/compilation/transpile_to_native_gates.py

```diff
@@ -210,7 +211,7 @@ def preprocess_gate(gate):
         return reversed(gate_operation_decomposition)
 
 
-class CCZtoT(DecompositionRule[GateOperation]):
+class CCXtoT(DecompositionRule[GateOperation]):
     """Decomposition of Toffoli into T, H, and CNOT gates."""
 
     def predicate(self, operation: GateOperation) -> bool:
```

Comment:
CCZ should be CCX in comment on next line.

Reply:
Thanks!